src/txn/txn_recover.c

```diff
@@ -449,6 +452,9 @@ __recovery_setup_file(WT_RECOVERY *r, const char *uri, const char *config)
       (WT_IS_MAX_LSN(&r->max_ckpt_lsn) || __wt_log_cmp(&lsn, &r->max_ckpt_lsn) > 0))
         r->max_ckpt_lsn = lsn;
 
+    /* Update the base write gen based on this file's configuration. */
+    WT_RET(__wt_metadata_update_base_write_gen(r->session, config));
+
```

Comment:
This can just be a single `return` statement, we don't need `WT_RET`.

Reply:
Fixed